packages/template-retail-react-app/app/components/_app/index.test.js

```diff
@@ -89,7 +104,22 @@ describe('App', () => {
     })
 
     test('Active Data component is not rendered', async () => {
-        constants.ACTIVE_DATA_ENABLED = false
+        mockActiveDataFlag.mockImplementation(() => false)
```

Comment:
This refactor addressed the eslint error
![image](https://github.com/user-attachments/assets/8cec6952-2488-4a68-b24e-bddb60c10f3a)